src/FSharp.Data.GraphQL.Server.AspNetCore/Giraffe/HttpHandlers.fs

```diff
@@ -270,12 +257,12 @@ module HttpHandlers =
         }
 
         taskResult {
-            let executor = Schema.executor
+            let executor = options.SchemaExecutor
             ctx.Response.Headers.Add("Request-Type", "Classic") // For integration testing purposes
```

Comment:
this can't remain here; as the comment says: it's for integration testing purposes only (or not?)... That's why I included that function that allowed for customizing the response header from outside of this code.

Reply:
Yes, it must be moved to samples project

Reply:
Removed